Swift/BRCrypto/BRCryptoSystem.swift

```diff
@@ -1166,6 +1166,162 @@ extension System {
     }
 }
 
+extension System {
```

Comment:
Is this really generic or is it BTC/BCH specific? The fields seem blockchain-specific, or are these all found in other blockchains?

Reply:
It is BTC/BCH specific - because that is the only existing persistent data that needs to be migrated.  The code reads 'more generically' then is should.  Perhaps something like `TransferBlob` needs a BTC/BCH specific name or another approach altogether?

```
enum TransferBlob { 
    case btc ((bytes: [UInt8], blockHeight: UInt32, timestamp: UInt32))
}
```

Reply:
That's an interesting idea; I kind of like that.

Reply:
Let me see how it plays out.

Reply:
Probably renaming `cryptoWalletMigratorHandleTransaction()` to `cryptoWalletMigratorHandleTransactionAsBTC()` too?

Reply:
I guess so, and block? If this is more trouble than its worth, we can revisit later.